bedrock/releasenotes/tests/test_base.py

```diff
@@ -358,10 +358,10 @@ def test_desktop_beta_version(self):
                    '/firefox/23.0beta/releasenotes/')
 
     @patch('bedrock.releasenotes.views.firefox_desktop.latest_version',
-           Mock(return_value='24.0a2'))
+           Mock(return_value='23.0b1'))
     def test_desktop_developer_version(self):
         self._test('/firefox/developer/notes/',
-                   '/firefox/24.0a2/auroranotes/')
+                   '/firefox/23.0beta/releasenotes/')
```

Comment:
I just meant keeping a test for a `/firefox/xx.0ax/auroranotes/` url (as it's being removed in this change).